.aws/src/monitoring.ts

```diff
@@ -167,7 +167,7 @@ export class RecommendationApiSynthetics extends Construct {
               'RECOMMENDATION_API_DOMAIN': config.domain,
           }
         },
-        runtimeVersion: 'syn-python-selenium-1.3',
+        runtimeVersion: 'syn-python-selenium-4.1',
```

Comment:
1.3 has been deprecated back in March. https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_python_selenium.html